test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5518,7 +5527,10 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
                self.attributes[k] != self.dflt_attributes[k]):
                 setdict[k] = self.dflt_attributes[k]
         if delhooks:
-            reverthooks = False
+            if self.platform == 'cray' or self.platform == 'craysim':
```

Comment:
I don't think we need this code as user or admin is not allowed to change PBS_translate_mpp hook.